src/status_im/accounts/create/core.cljs

```diff
@@ -214,7 +219,11 @@
                                          (inc-step step))}
                           (when (and (= step :confirm-code)
                                      (not (:accounts/login db)))
-                            (create-account cofx))))))
+                            (if recovery?
+                              (accounts-recover-recover-account
+                               (security/safe-unmask-data (get-in db [:intro-wizard :masked-passphrase]))
```

Comment:
Would be better to extract `masked-passphrase` and `key-code` in a `let` binding together with `step` for better readability.

Reply:
Done!